test/controllers/user.test.js

```diff
@@ -11,7 +14,23 @@ const userAttributes = {
   password: 'password1923'
 };
 
-const createUser = () => request.post('/users').send(userAttributes);
+const signUpUser = () => request.post('/users').send(userAttributes);
+const createAndSignInUser = () => {
+  userAttributes.password = 'password1923';
+  return signUpUser().then(() =>
+    request
+      .post('/users/sessions')
+      .send({ email: 'omar.rodriguez@wolox.com', password: 'password1923' })
+      .then(response => response.body.response)
+  );
+};
+
+factory.define('user', models.user, {
+  firstName: faker.name.firstName,
+  lastName: faker.name.lastName,
+  email: faker.internet.email,
+  password: bcrypt.hashSync('password1923', 10)
+});
```

Comment:
What is the purpose of this? You already have thee user defined in the factory file.